backends/saves/windows/windows-saves.cpp

```diff
@@ -32,15 +32,25 @@
 
 #include "common/scummsys.h"
 #include "common/config-manager.h"
+#include "common/file.h"
 #include "backends/saves/windows/windows-saves.h"
 #include "backends/platform/sdl/win32/win32_wrapper.h"
 
+#define DEFAULT_CONFIG_FILE "scummvm.ini"
```

Comment:
It might be a good idea to move all definitions of `DEFAULT_CONFIG_FILE` to win32.h or a similar header to reduce duplication.